SMLHelper/Handlers/IngameMenuHandler.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace SMLHelper.V2.Handlers
+{
+    using System;
+    using SMLHelper.V2.Interfaces;
+    using SMLHelper.V2.Patchers;
+
+    /// <summary>
+    /// A handler class that offers simple ways to tap into functionality of the in game menu.
+    /// </summary>
+    public class IngameMenuHandler : IIngameMenuHandler
+    {
+        /// <summary>
+        /// Main entry point for all calls to this handler.
+        /// </summary>
+        public static IIngameMenuHandler Main { get; } = new IngameMenuHandler();
+
+        private IngameMenuHandler() { } // Hide constructor
+
+        /// <summary>
+        /// Registers a simple <see cref="Action"/> method to invoke whenever the player saves the game via the in game menu.
+        /// </summary>
+        /// <param name="onSaveAction">The method to invoke.</param>
+        public static void RegisterOnSaveEvent(Action onSaveAction)
+        {
+            Main.RegisterOnSaveEvent(onSaveAction);
+        }
+
+        /// <summary>
+        /// Removes a method previously added through <see cref="RegisterOnSaveEvent(Action)"/> so it is no longer invoked when saving the game.<para/>
+        /// If you plan on using this, do not register an anonymous method.
+        /// </summary>
+        /// <param name="onSaveAction">The method invoked.</param>
+        public static void UnregisterOnSaveEvent(Action onSaveAction)
+        {
+            Main.UnregisterOnSaveEvent(onSaveAction);
+        }
+
+        /// <summary>
+        /// Registers a simple <see cref="Action"/> method to invoke whenever the player saves the game via the in game menu.
+        /// </summary>
+        /// <param name="onSaveAction">The method to invoke.</param>
```

Comment:
The summary needs to reflect the fact that this method is one-time. Also, I think you could merge `RegisterOnSaveEvent` with `RegisterOneTimeUseOnSaveEvent`, and have an optional `bool` parameter that defaults to `false` dictating whether that event is one-time or not.

Reply:
I considered merging the two methods with an optional parameter, but then it complicated the documentation and explanation of the `UnregisterOnSaveEvent` method.

To keep things simple, I thought it best to just make a separate method with clear documentation.